census/templates/profile/_blocks/_stat_list.html

```diff
@@ -3,7 +3,7 @@
     <span class="{% if stat_class %}{{ stat_class }}{% else %}primary{% endif %}">
         <span class="value">
             {% if not stat.values.this %}
-                N/A
+                {% if WAZIMAP.na_label %}{{ WAZIMAP.na_label }}{% else %}N/A{% endif %}
```

Comment:
Instead of the if/else, put a default setting into wazimap/settings.py and just use WAZIMAP.na_label.